docs/api/ProxyAgent.md

```diff
@@ -19,7 +19,10 @@ Extends: [`AgentOptions`](Agent.md#parameter-agentoptions)
 * **uri** `string` (required) - It can be passed either by a string or a object containing `uri` as string.
 * **token** `string` (optional) - It can be passed by a string of token for authentication.
 * **auth** `string` (**deprecated**) - Use token.
-* **clientFactory** `(origin: URL, opts: Object) => Dispatcher` - Default: `(origin, opts) => new Pool(origin, opts)`
+* **clientFactory** `(origin: URL, opts: Object) => Dispatcher` (optional) - Default: `(origin, opts) => new Pool(origin, opts)`
+* **requestTls** `BuildOptions` (optional) - creates the underlying socket via the connector builder
+* **proxyTls** `BuildOptions` (optional) - creates the underlying socket via the connector builder
+* **factory** `(_origin: URL, opts: Object) => Dispatcher` (optional) - Default: `(_origin: URL, opts: Object) => new Agent(opts)`
```

Comment:
I believe `factory` is reflecting `clientFactory`, but the options only accepts `clientFactory`